national-insurance.conf

```diff
@@ -6055,8 +6055,10 @@
       year = £50270
     }
     DPT {
-      week = £229
-      year = £11908
+      week = £242
+      four-week = £967
```

Comment:
Where did the four-week and month options come from - can't see them in the ticket
Historically the only entries for DPT are week and year

Reply:
DPT has now been removed as there is no separate directors section